email/1.3/modules/ROOT/pages/email-trigger.adoc

```diff
@@ -0,0 +1,219 @@
+= Trigger a Flow When Receiving a New Email 
+:page-aliases: connectors::email/email-trigger.adoc
+
+Anypoint for Email (Email Connector) provides the *On New Email - IMAP* and *On New Email - POP3* source listeners that trigger a flow by retrieving an email folder from a mailbox for all the unread emails, and generate a new message for each email found. The following examples illustrates how to configure these sources parameters to identify a new email when triggering a flow.
+
+== Configure Delete After Retrieve Parameter
+
+Email Connector *On New Email - IMAP* and *On New Email - POP3* sources support the *Delete after retrieve* parameter that enables the deletion of retrieved emails from the mailbox folder after the operation process is complete. By deleting processed emails, you can ensure that the next processed emails are new ones. Because the POP3 protocol doesn't support watermarking, the *Delete after retrieve* parameter is commonly used with this protocol. +
+The parameter is disabled by default to avoid deleting emails by mistake.
+
+The following example shows how to configure this parameter in Anypoint Studio:
+
+. Select the *On New Email - IMAP* or *On New Email - POP3* source in the Studio canvas.
+. In the source configuration screen, enable the *Delete after retrieve* parameter.
+
+The following screenshot shows the *Delete after retrieve* configuration for the *On New Email - IMAP* source:
+
+.Delete after retrieve with On New Email - IMAP operation
+image::email-delete-1.png[Delete after retrieve with On New Email - IMAP]
+
+In the XML editor, the configuration `deleteAfterRetrieve="true"` is under the `listener-imap` tag:
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-IMAP">
+		<email:listener-imap config-ref="Email_IMAP" watermarkEnabled="false" deleteAfterRetrieve="true">
+			<scheduling-strategy >
+				<fixed-frequency />
+			</scheduling-strategy>
+		</email:listener-imap>
+	</flow>
+----
+
+The following screenshot shows the *Delete after retrieve* configuration for the *On New Email - POP3* source:
+
+.Delete after retrieve with On New Email - POP3 operation
+image::email-delete-2.png[Delete after retrieve with On New Email - POP]
+
+In the XML editor, the configuration `deleteAfterRetrieve="true"` is under the `listener-pop3` tag:
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-POP3">
+  <email:listener-pop3 config-ref="Email_POP3" deleteAfterRetrieve="true">
+    <scheduling-strategy >
+      <fixed-frequency />
+    </scheduling-strategy>
+  </email:listener-pop3>
+</flow>
+----
+
+
+== Configure Enable Watermark Parameter
+
+Email Connector can watermark emails over the IMAP protocol by filtering emails based on their received date. The *On New Email - IMAP* source supports the *Enable Watermark* parameter that enables tp dispatch only messages containing emails received after the last poll executed.
```

Comment:
tp dispatch? TP dispatch?  I'm not sure. I don't know what tp dispatch enabled means.